src/components/Home.js

```diff
@@ -81,100 +85,114 @@ const Home = () => {
     }
   }
 
-  const transactions = [{
-    name: 'Netflix',
-    price: 20,
-    date: '12/04/2020',
-    category: 'Entertainment'
-  }, {
-    name: 'Amazon',
-    price: 120,
-    date: '12/04/2020',
-    category: 'Miscellaneous'
-  }, {
-    name: 'Samsung',
-    price: 230,
-    date: '11/04/2020',
-    category: 'Bill'
-  }, {
-    name: 'McDonalds',
-    price: 15.99,
-    date: '09/04/2020',
-    category: 'Food & Drink'
-  }, {
-    name: 'Tesco',
-    price: 175.85,
-    date: '08/04/2020',
-    category: 'Grocery'
-  }, {
-    name: 'Travel expenses',
-    price: 400,
-    date: '07/04/2020',
-    category: 'Holiday'
-  }, {
-    name: 'Asos',
-    price: 130,
-    date: '01/04/2020',
-    category: 'Shopping'
-  }, {
-    name: 'Asda',
-    price: 23,
-    date: '19/03/2020',
-    category: 'Grocery'
-  }, {
-    name: 'Uniqlo',
-    price: 90.50,
-    date: '18/03/2020',
-    category: 'Shopping'
-  }, {
-    name: 'Thames Water',
-    price: 90,
-    date: '10/03/2020',
-    category: 'Bill'
-  }, {
-    name: 'Louis Vuitton',
-    price: 1300,
-    date: '28/02/2020',
-    category: 'Shopping'
-  }, {
-    name: 'Dyson',
-    price: 200,
-    date: '27/02/2020',
-    category: 'Shopping'
-  }, {
-    name: 'Chanel',
-    price: 2650,
-    date: '27/02/2020',
-    category: 'Shopping'
-  }, {
-    name: 'Taxi',
-    price: 12.50,
-    date: '27/02/2020',
-    category: 'Transport'
-  }, {
-    name: 'Jason',
-    price: 36,
-    date: '20/02/2020',
-    category: 'Person'
-  }, {
-    name: 'Dentist',
-    price: 250,
-    date: '19/02/2020',
-    category: 'Personal Care'
-  }, {
-    name: 'Vitamins',
-    price: 12,
-    date: '18/02/2020',
-    category: 'Personal Care'
-  }, {
-    name: 'Unicef',
-    price: 20,
-    date: '17/02/2020',
-    category: 'Charity'
-  }]
+  // const transactions = [{
```

Comment:
@tams2429 can we get rid of these commented out blocks? Generally its not a good idea to check these into source control - it messes with the diffs and makes it harder to see exactly whats changed